tests/src/test/java/io/streamnative/pulsar/handlers/kop/compatibility/saslplain/SaslPlainEndToEndTest.java

```diff
@@ -265,7 +265,6 @@ void testKafkaProduceAndConsumeWithSaslPlain() throws Exception {
 
             // ensure that we can list the topic
             Map<String, List<PartitionInfo>> result = consumer.listTopics(1000);
-            assertEquals(result.size(), 1);
```

Comment:
The same question as I asked in previous comment. Why did you remove this check?